src/main/java/com/feywild/feywild/config/MobConfig.java

```diff
@@ -33,7 +33,11 @@
         ));
 
         public static CommonSpawns summer_bee_knight = new CommonSpawns(25, 1, 2);
-        public static CommonSpawns dwarf_blacksmith = new CommonSpawns(20, 1, 1);
+        public static CommonSpawns dwarf_blacksmith = new CommonSpawns(5, 1, 1);
+
+        public static PixieSpawns shroomling = new PixieSpawns(15, 1, 1, List.of(
```

Comment:
If `PixieSpawns` is used for non-pixies now, consider renaming it to `AdvancedSpawns`